frappe/model/base_document.py

```diff
@@ -365,14 +365,16 @@ def _get_table_fields(self):
 	def get_valid_dict(
 		self, sanitize=True, convert_dates_to_str=False, ignore_nulls=False, ignore_virtual=False
 	) -> _dict:
+		from frappe.model.document import DocRef
+
 		d = _dict()
 		field_values = self.__dict__
 
 		for fieldname in self.meta.get_valid_fields():
-			value = field_values.get(fieldname)
```

Comment:
What was the reason for renaming this throughout the function?

Reply:
I found this pattern in the very nicely written query engine and liked it. No need, I'll revert to reduce the diff.

It's a nice way to communicate: this is a local, temporary value that is being worked on, i.e. can change its type.

Reply:
Ah, okay.

Yeah maybe can have a separate PR with these kinda of refactors, maybe other places in the codebase could reuqire similar changes, can exclude that from git blame then.